inspector/src/lod.ts

```diff
@@ -2,12 +2,17 @@ import { Effect } from "babylonjs/Materials/effect";
 
 let name = 'lodPixelShader';
 let shader = `
+#ifdef GL_ES
```

Comment:
this is not usefull in webgl GL_ES is always true :-)

Reply:
Ah, do we need to update the example from this doc then? https://doc.babylonjs.com/how_to/how_to_use_postprocesses#custom-postprocesses

Reply:
would be great

Reply:
PR created